vocabulary/index.html

```diff
@@ -4679,7 +4679,7 @@ <h2>Properties</h2>
               The content or textual representation of the Object encoded as a
               JSON string. By default, the value of <code>content</code> is
               HTML. The <code><a>mediaType</a></code> property can be used in
-              the object to indicate a different content type.
+              the object to indicate a different media type.
```

Comment:
this one is tricky because it's the media type of the `content` string, not the object itself. I would say "content media type" for clarity.

Reply:
Clarified in https://github.com/w3c/activitystreams/pull/639/commits/ff56515911c0015b03ff5e4c46bccde86fed72d1